x/taskmanageravs/client/cli/tx.go

```diff
@@ -0,0 +1,76 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+
+	// "github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/ExocoreNetwork/exocore/x/taskmanageravs/types"
+	taskTypes "github.com/ExocoreNetwork/exocore/x/taskmanageravs/types"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+)
+
+var (
+	DefaultRelativePacketTimeoutTimestamp = uint64((time.Duration(10) * time.Minute).Nanoseconds())
+)
+
+const (
+	flagPacketTimeoutTimestamp = "packet-timeout-timestamp"
+	listSeparator              = ","
+)
+
+// GetTxCmd returns the transaction commands for this module
+func GetTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      fmt.Sprintf("%s transactions subcommands", types.ModuleName),
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	// this line is used by starport scaffolding # 1
+	cmd.AddCommand(
+		CreateTask(),
+	)
+	return cmd
+}
+
+func CreateTask() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "Set task params to taskManager module",
+		Short: "Set task params to taskManager module",
+		Args:  cobra.ExactArgs(2),
```

Comment:
The exact number of arguments should be 4.

Reply:
task manager will be handled in https://github.com/ExocoreNetwork/exocore/pull/20